lib/core/network/api/aws_amplify/amplify_service.dart

```diff
@@ -156,14 +159,18 @@ class AmplifyService {
       print('logged in: ${session?.isValid()}');
 
       return true;
-    } on CognitoUserConfirmationNecessaryException catch (e) {
-      print('CognitoUserConfirmationNecessaryException $e');
+    } on CognitoUserConfirmationNecessaryException catch (error, stackTrace) {
+      LogHelper.e(
```

Comment:
If there was no error message, we should keep it this way

The way cognito login works is:
login with password, this fails, then log in with necessary confirmation, so there is an exception, but it's expected and should not be logged, since it happens on every single login.

Reply:
So yeah I am pretty sure this is intentional, this should not have an error message, and definitely not log it on crashlytics.